server/src/main/java/org/elasticsearch/index/translog/MultiSnapshot.java

```diff
@@ -56,6 +58,11 @@ public int totalOperations() {
         return totalOperations;
     }
 
+    @Override
+    public int skippedOperations() {
```

Comment:
java docs please.

Reply:
javadoc is in Snapshot - shall it be copied here ?

Reply:
sorry, I may be missing something. Where in snapshot?

Reply:
I see. I looked at TranslogSnapshot. All good then. Thanks.